contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -21,105 +21,111 @@ If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fa
+But, as I said, that was *two years ago*. Today, Clickhouse has broken ground on features that compete with Snowflake’s managed solution. Likewise, Snowflake released features that are beginning to tackle ClickHouse’s speed. However, the overlap between ClickHouse and Snowflake remains nascent. Today, they are both still  *very, very* different.
 
 ## Background
 
 Snowflake and ClickHouse are at wildly different stages of growth.
 
 Snowflake, founded in 2012, is a well-funded, publicly-traded decacorn with nearly 4,000 employees.
 
-In contrast, ClickHouse is relatively new as an independent company. While the project originated internally at Yandex around 2012, ClickHouse only spun out independently in 2016 and received funding in 2021. Since then it has been growing rapidly. 
+In contrast, ClickHouse is in relative infancy. While the project originated internally at Yandex around 2012, ClickHouse only spun out independently in 2016 and received funding in 2021. 
 
 Recently, ClickHouse raised a Series B to launch ClickHouse Cloud, its Snowflake-like service.
 
 ### What even is Snowflake?
 
-Every engineer and their cousin has heard about Snowflake. Its massive IPO was one of the most successful public launchs in technology history. The title of “Next Snowflake” translates to “Next Incredible Business” in venture-capital land. And with over $1B in revenue, the hype somewhat deserved. However, there’s also a rather curious phenomenon – relatively speaking, few engineers appear to be versed in SnowSQL (Snowflake’s SQL dialect).
+Every engineer and their cousin has heard about Snowflake. Its massive IPO was the most successful public launch in technology history. The title of “Next Snowflake” translates to “Next Incredible Business” in venture-capital land. And with over $1B in revenue, the hype is deserved. However, there’s also a rather curious phenomenon – relatively speaking, few engineers appear to be versed in SnowSQL (Snowflake’s SQL dialect).
 
-This contradiction between success and adoption is because Snowflake doesn’t behave like a typical database company. Snowflake is closed-source and cloud-only, unlike solutions like ClickHouse, [Apache Druid](/blog/clickhouse-vs-druid), TimescaleDB, QuestDB, and most other OLAP databases. And sorry, startups, but there is no free tier. Go directly to a paid plan, do not pass go, do not collect $200.
+This contradiction between success and adoption is because Snowflake doesn’t behave like a typical database company. Snowflake is cloud-only, unlike solutions like ClickHouse, [Apache Druid](/blog/clickhouse-vs-druid), TimescaleDB, and most other OLAP databases. The codebase is neither open source nor locally installable. And sorry, startups, but there is no free tier.
```

Comment:
```suggestion
This contradiction between success and adoption is because Snowflake doesn’t behave like a typical database company. Unlike ClickHouse, [Apache Druid](/blog/clickhouse-vs-druid), TimescaleDB, QuestDB, and most other OLAP databases Snowflake is cloud-only. The codebase is neither open source nor locally installable. And sorry, startups, but there is no free tier.
```